src/widgets/Xpert.jsx

```diff
@@ -29,27 +40,36 @@ const Xpert = ({
     dispatch(updateSidebarIsOpen(isOpen));
   };
 
-  useEffect(() => {
-    dispatch(getLearningAssistantChatSummary(courseId));
-  }, [dispatch, courseId]);
+  const [decision] = useAuditTrialExperimentDecision();
+  const { enabled, variationKey } = decision || {};
+
+  const shouldDisplayXpert = () => (
+    // if a user is not part of the experiment, not part of the control, or not eligible for upgrade, they should
+    // still see xpert
+    !enabled
+    || !(variationKey === OPTIMIZELY_AUDIT_TRIAL_LENGTH_EXPERIMENT_VARIATION_KEYS.CONTROL)
+    || isStaff
+  );
```

Comment:
NIT: Just a suggestion. I'd rather rename `enabled` to `experimentEnabled`, just to have more context. Ignore this if you don't agree.
```suggestion
  const { enabled: experimentEnabled, variationKey } = decision || {};

  const shouldDisplayXpert = () => (
    // if a user is not part of the experiment, not part of the control, or not eligible for upgrade, they should
    // still see xpert
    !experimentEnabled
    || !(variationKey === OPTIMIZELY_AUDIT_TRIAL_LENGTH_EXPERIMENT_VARIATION_KEYS.CONTROL)
    || isStaff
  );
```